Makefile

```diff
@@ -30,6 +31,12 @@ test-global-dependency: tests/test-global-dependency.c
 	$(CC) -fPIC -c -emit-llvm $< -o test.bc
 	./split-llvm-extract test.bc -o out
 
+test-extern-const-constptr: $(wildcard tests/test-extern-const-constptr/*.c)
```

Comment:
I think we need to link this into the build? `all: ... test-extern-const-constptr` or similar? Also, do we want to run the resulting binary or is that not necessary to check the property we care about?

Reply:
The other tests are not there because [lang-tester](https://github.com/capablevms/llvm-function-split/blob/d82acbf76a91e4e7a0460be5e0afe2bb4eb10396/tests/run.rs#L66) checks the outputs at the begging of buildbot.sh if I am not wrong. I haven't got the chance of learning Rust yet but I do not think it requires any modification as long as I follow the convention in the C code comments.